google/cloud/spanner/commit_options.h

```diff
@@ -70,6 +71,16 @@ class CommitOptions {
     return request_priority_;
   }
 
+  CommitOptions& set_max_batching_delay(
+      absl::optional<absl::Duration> max_batching_delay) {
```

Comment:
We should not use `absl::Duration` in a public API.

Reply:
Done, changed to type int and renamed to `max_batching_delay_ms`.

Reply:
Sorry for the confusion.  As @coryan pointed out, "should not use `absl::Duration` in a public API" means to use `std::chrono::duration` instead.